### ISSUE TITLE
Have You Ever Heard of Default Values?

### ISSUE THREAD

**Issue by machineghost** [moroshko/react-autosuggest#694]
## Are you reporting a bug?

* Provide the steps to reproduce the issue, e.g.:
  1. Try to use library
  2.  Give up because there's so much pointless boilerplate

  **Observed behaviour:** Suggestions stay open
See above.

  **Expected behaviour:** Suggestions should be closed
It's an autocomplete widget!!! It should _require_ exactly **one** prop: a list of values to complete from, or a function to retrieve those values.
## Are you making a feature request?

* Please describe your use case from user journey point of view, e.g.:

When I come to a library, I want to get shit done, not read an 18 hour treatise on how to configure it with umpteen different props.  A good library _will_  have every option I could possibly want available ... but it won't force me to learn and use every last one just use to use the library.

It's an incredibly frustrating user experience when **ANY** is clearly written by someone very smart ... who seems to have thrown all intelligence out the window when it came to setting sensible defaults and letting people start using a library without taking a master course in how to do so.

**Comment by aberezkin** [comment]
I don't think that this is a library for a "narrow" use case. It's low level so anyone can implement the behavior they want to see in their UX scenarios. It's the opposite of narrow. Most of the issues in this repo in fact are asking about more customizable elements instead of less.

**Comment by machineghost** [comment]
>I don't think that this is a library for a "narrow" use case.

Exactly my point (that question was meant to be more rhetorical).

If it's intended to be used by anyone, then philosophically one would expect such a library to *require* as little as possible.  But (and this was really the core of the issue) the library *unnecessarily requires* that things be provided, when they could instead be optional defaults (which could of course be overridden based on user needs).

**Comment by moroshko** [comment]
At this point, we expose a single `Autosuggest` component, which is a [controlled component](https://reactjs.org/docs/forms.html#controlled-components). You can think about it as a low level API. 

We could definitely explore exposing a higher level components (probably uncontrolled), that would also have some sensible defaults like you are suggesting.

**Comment by rafalkrupinski** [comment]
+1 to the sensible defaults. Currently the basic case in Readme is ca. 100 LOCs, it should be 20

**Comment by machineghost** [comment]
>At this point, we expose a single Autosuggest component, which is a controlled component. You can think about it as a low level API.

>We could definitely explore exposing a higher level components (probably uncontrolled), that would also have some sensible defaults like you are suggesting.

This doesn't make any sense to me.  In what way does setting default `props` require a new component?
